kernel-crawler/Makefile

```diff
@@ -264,7 +264,7 @@ crawl-ubuntu-standard: build-crawl-container
 
 .PHONY: crawl-minikube
 crawl-minikube: build-crawl-container
-	docker run --rm -i kernel-crawler crawl Minikube > $(CRAWLED_PACKAGE_DIR)/minikube.txt
```

Comment:
Is the entrypoint redundant here since `python3` is executed by the shebang in `minikube-crawler.py`?

Reply:
I think you are right, I copied this from the garden linux crawler which might not have the shebang. I'll give it a try.

Reply:
Actually, now I remember why that is needed, the dockerfile sets the entrypoint to `python3 kernel-crawler.py` ([here](https://github.com/stackrox/kernel-packer/blob/60ffc66a546f8b62e4e52c4e3e4b9c30c2ab66ca/kernel-crawler/Dockerfile#L33)), so we need to override that when running other custom crawlers.